docs/patterns/components/TwoColumnPanel/index.js

```diff
@@ -126,6 +126,10 @@ function TwoColumnPanelDemo() {
               Lorem ipsum dolor sit amet, consectetur adipiscing elit.
               Pellentesque efficitur sollicitudin ornare.
             </p>
+            <Code
+              role="region"
+              tabIndex="0"
+            >{`<div>example of an element that has horizontal scrolling</div>`}</Code>
```

Comment:
I don't think we necessarily need an example here of an element that scrolls (although it did make it easier for me to verify).

Reply:
Yeah, I had a similar thought, but wanted to leave it in for verification. Happy to pull out, though.